android/spindance.code-challenge.mobile.android.weather-sensor-service/LICENSE.md

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2022 SpinDance
```

Comment:
License appears twice? Is that related to the submodules thing?

Reply:
Hum maybe, I'll fix that